README.md

```diff
@@ -117,10 +115,16 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc
+$ ./x.py dist --build=x86_64-pc-windows-msvc --install
```

Comment:
Why is it no longer relevant? MSYS2 is still the best environment to work with both GNU and MSVC flavors.
<sub><sup>Because PowerShell is goddamn horrible.</sup></sub>